bundle/manifests/grafana-operator.clusterserviceversion.yaml

```diff
@@ -559,7 +559,11 @@ spec:
       name: manager
     - image: ghcr.io/grafana/grafana-operator@sha256:97561cef949b58f55ec67d133c02ac205e2ec3fb77388aeb868dacfcebad0752
       name: grafana-operator-97561cef949b58f55ec67d133c02ac205e2ec3fb77388aeb868dacfcebad0752-annotation
-  replaces: grafana-operator.v5.6.2
+  replaces: grafana-operator.v5.6.0
```

Comment:
How will this affect the replacement of 5.8.0? Will the 5.8.0 users get 5.8.1 as well?

Reply:
Yes, all versions specified in the `skips` block will update to 5.8.1, including 5.8.0. It's based on this concept: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/how-to-update-operators.md#skipping-updates